lib/src/session-replay-browser.ts

```diff
@@ -2,6 +2,13 @@ import type * as SessionReplay from '../generated/browserSessionReplay'
 
 export * from '../generated/browserSessionReplay'
 
+export const BrowserSource: (undefined | SessionReplay.BrowserSegmentMetadata['source'])[] = [
+  /**
+   * For backward compatibility reasons, `undefined` should be accepted as a possible value for Browser segments, too.
+   **/
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏did you intend to add `undefined` to this array then? Or is it something to check in web-ui?